lib/puppet/type/acl.rb

```diff
@@ -184,7 +195,7 @@ def insync?(is)
         s = p.tr '-', ''
         r << (s.sub!('r', '')?'r':'-')
         r << (s.sub!('w', '')?'w':'-')
-        r << (s.sub!('x', '')?'x':'-')
+        r << (s.sub!('x', '')?'x':'-')#'
```

Comment:
Looks like the ternary operator needs spaces to be digestible to other syntax highlighters as well: 
![bad_ternary_parsing](https://cloud.githubusercontent.com/assets/1537044/9937192/ab7d7608-5d2d-11e5-9bce-9ef21102ee93.png)